roqollage/src/interface.rs

```diff
@@ -798,6 +805,7 @@ pub fn add_gate(
                 op.condition_register(),
                 op.condition_index(),
             ));
+            let group_index = circuit_gates[min].len() - 1;
```

Comment:
Does this correspond to old_len[min]?

Reply:
Not exactly since old_len[min] was wrong, old_len contains the length that is shown on the image (so it ignores some gates that are not diplayed) while group index is the exact index of the gate_group in circuit_gates